numato_relay/package.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>numato_relay</name>
+  <version>0.0.0</version>
```

Comment:
Can you please bump the version to 2.0.0 for ROS 2?

Reply:
Change it to the old version and blooming will bump the major.

Reply:
The state of _package.xml_ in the main branch is currently ` <version>0.1.3</version>`
https://github.com/clearpathrobotics/numato_relay/blob/fe7e89483f0d8b7284b4219a924b5b60a9ac0970/package.xml#L4C12-L4C18

@tonybaltovski are you suggesting that I change the Humble branch to also have the version as  `<version>0.1.3</version>`?

Reply:
Correct.